webpack.config.js

```diff
@@ -1,38 +1,29 @@
 var webpack = require( 'webpack' ),
 	path = require( 'path' ),
+	CleanWebpackPlugin = require( 'clean-webpack-plugin' ),
 	ExtractTextPlugin = require( 'extract-text-webpack-plugin' );
 
-// var dependencies = [
-// 	'react',
-// 	'react/addons',
-// 	'babelify/polyfill'
-// ];
-
 // TODO autoprefixer:
 // browsers: ['> 1%', 'last 2 versions', 'ff 17', 'opera 12.1', 'ie 8', 'ie 9', 'safari 7', 'safari 8'],
 // map: true
 
 var IS_HOT_UPDATE = ( process.env.NODE_ENV !== 'production' );
 
-var styleDefaults = 'css?sourceMap!autoprefixer!sass?sourceMap';
+var scssLoader = [ 'style-loader', 'css-loader', 'autoprefixer-loader', 'sass-loader' ];
```

Comment:
Noticed a build error after I finished my review -- it sounds like `style-loader` doesn't work with `ExtractTextPlugin` except as a fallback. Once I removed it from this array it built fine.